generate-node-metrics.sh

```diff
@@ -0,0 +1,120 @@
+function exit_handler()
```

Comment:
this will be most probably taken care by `elastic-package benchmark rally` (https://github.com/elastic/elastic-package/issues/1475, https://github.com/elastic/elastic-package/issues/984#issuecomment-1733191971)